src/legacy/core_plugins/timelion/public/plugin.ts

```diff
@@ -35,8 +29,7 @@ import { LegacyDependenciesPlugin, LegacyDependenciesPluginSetup } from './shim'
 
 /** @internal */
 export interface TimelionVisualizationDependencies extends LegacyDependenciesPluginSetup {
-  uiSettings: IUiSettingsClient;
-  http: HttpSetup;
+  core: CoreSetup<PluginsStart>;
```

Comment:
Please create a `TimelionPluginStartDependencies` for this generic type:
```ts
interface TimelionPluginStartDependencies {
  data: DataStart;
}
```

Reply:
Don't have a necessity in it anymore